README_OS_X.md

```diff
@@ -92,11 +91,11 @@ Once those dependencies are satisfied, the following steps will build Supercolli
 	$>	cd supercollider
 	$>	mkdir build; cd build
 	$>	cmake -G Xcode -DCMAKE_PREFIX_PATH=`brew --prefix qt5`  ..
-  $>  cmake --build . --target install --config Debug
+	$>	cmake --build . --target install --config Debug
```

Comment:
could we change this to RelWithDebInfo ?  its very likely that this example gets copypasted.

then its ready for merge I think